From a46be65ee8934fda6bff3057bdc989662ceda9a6 Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Wed, 22 Jun 2011 00:18:40 +0800
Subject: ENGR00151494-2 arch: change mxc_vpu.h to remove definition for
 mx32/mx37

The vpu code in driver for mx32/mx37 is out of date already, and
we won't support the platforms, so clean the code for removing it.

And removed ioctl definition that won't use again.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 arch/arm/plat-mxc/include/mach/mxc_vpu.h |    9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/arch/arm/plat-mxc/include/mach/mxc_vpu.h b/arch/arm/plat-mxc/include/mach/mxc_vpu.h
index 19c01b2..2dbe749 100755
--- a/arch/arm/plat-mxc/include/mach/mxc_vpu.h
+++ b/arch/arm/plat-mxc/include/mach/mxc_vpu.h
@@ -49,12 +49,9 @@ struct vpu_mem_desc {
 #define VPU_IOC_WAIT4INT	_IO(VPU_IOC_MAGIC, 2)
 #define VPU_IOC_PHYMEM_DUMP	_IO(VPU_IOC_MAGIC, 3)
 #define VPU_IOC_REG_DUMP	_IO(VPU_IOC_MAGIC, 4)
-#define VPU_IOC_VL2CC_FLUSH	_IO(VPU_IOC_MAGIC, 5)
 #define VPU_IOC_IRAM_SETTING	_IO(VPU_IOC_MAGIC, 6)
 #define VPU_IOC_CLKGATE_SETTING	_IO(VPU_IOC_MAGIC, 7)
 #define VPU_IOC_GET_WORK_ADDR   _IO(VPU_IOC_MAGIC, 8)
-#define VPU_IOC_GET_PIC_PARA_ADDR   _IO(VPU_IOC_MAGIC, 9)
-#define VPU_IOC_GET_USER_DATA_ADDR   _IO(VPU_IOC_MAGIC, 10)
 #define VPU_IOC_SYS_SW_RESET	_IO(VPU_IOC_MAGIC, 11)
 #define VPU_IOC_GET_SHARE_MEM   _IO(VPU_IOC_MAGIC, 12)
 
@@ -98,4 +95,10 @@ void vl2cc_flush(void);
 void vl2cc_disable(void);
 void vl2cc_cleanup(void);
 
+int vl2cc_init(u32 vl2cc_hw_base);
+void vl2cc_enable(void);
+void vl2cc_flush(void);
+void vl2cc_disable(void);
+void vl2cc_cleanup(void);
+
 #endif
-- 
1.7.9.5

